es-models/gene_expression/gene_expression.mapping.yaml

```diff
@@ -0,0 +1,31 @@
+_source:
+  excludes:
+  - genes.*
+properties:
+  age_at_diagnosis:
```

Comment:
I'm not sure which properties we'll really need, but I don't see any reason not to start with this set.

Reply:
I believe these are the ones that oncogrid uses.  So, that's what everyone asked for.